engines/python/setup/djl_python/deepspeed.py

```diff
@@ -122,6 +126,7 @@ def _parse_properties(self, properties):
             properties.get("tensor_parallel_degree", 1))
         self.low_cpu_mem_usage = properties.get("low_cpu_mem_usage",
                                                 "true").lower() == "true"
+        self.enable_streaming = bool(properties.get("enable_streaming", False))
```

Comment:
better call it streaming_token, this not not a real stream case

Reply:
`streaming_token` may not be suitable for cases when we support image models.